packages/moonstone/MoonstoneDecorator/MoonstoneDecorator.js

```diff
@@ -118,17 +120,113 @@ const MoonstoneDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	const Decorator = class extends React.Component {
 		static displayName = 'MoonstoneDecorator';
 
+		static contextTypes = contextTypes
+
+		static childContextTypes = contextTypes
+
+		static propTypes =  /** @lends moonstone/MoonstoneDecorator.AccessibilityDecorator.prototype */ {
```

Comment:
should we reference `moonstone/MoonstoneDecorator.AccessibilityDecorator.prototype`?